R/conf_mat_resampled.R

```diff
@@ -61,19 +60,17 @@ conf_mat_resampled <- function(x, ..., parameters = NULL, tidy = TRUE) {
       dplyr::select(!!!params) %>%
       distinct()
     if (nrow(param_combos) > 1) {
-      rlang::abort(
-        paste0(
-          "It looks like there are ", nrow(param_combos), " tuning parameter ",
-          "combination(s) in the data. Please use the `parameters` ",
-          "argument to select one combination of parameters."
-        )
+      cli::cli_abort(
+        "It looks like there are {nrow(param_combos)} tuning parameter
+        combination{?s} in the data. Please use the {.arg parameters}
```

Comment:
```suggestion
        combination in the data. Please use the {.arg parameters}
```

This error is gated by `nrow(param_combos) > 1`